database/open_source/blogs.json

```
@@ -26,5 +26,12 @@
     "url": "https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github",
     "category": "open source",
     "subcategory": "open-source-blogs"
+  },
+  {
+    "name": "Open Source With Pradumna",
+    "description": "Building the repo you need to get started with Open Source",
```

[Proofreading] This description could be clearer. What does this blog discuss? Try something like this:
```suggestion
    "description": "This blog provides tips and tricks to building the repo you need to start your journey in Open Source",
```

---

Ok i will do it today